### DISCUSSION TITLE
New permission

### DISCUSSION THREAD

**Discussion by JoanFo1456** [pelican-dev/panel#653] [Ideas]
This may sound dumb, but, create a "download" specific perm, because if you want to give access to someone to develop, in case, you would disable for him the downloads by removing that perm (if it's added), so you don't have to modify wings, etc, and disable fully the downloads for everyone.
(This idea was from a developer friend, and he didn't have the courage to post it here)

**Comment by rmartinoscar** [comment]
Hey I believe this is already a part of the legacy permission system, if you don't want them to be able to download a file uncheck the `read-content` & `sftp` permissions for that said user
![image](https://github.com/user-attachments/assets/01769a5c-5b3e-405f-b7e7-1b7a00ccceec)

**Comment by Boy132** [comment]
`read-content` is the (subuser) permission that controls downloads, like Martin already said.

**Comment by JoanFo1456** [comment]
If you do that, they cannot see the content on the file, i mean to separate the read-content into two
Read-Content AND Download

**Reply by rmartinoscar** [reply]
If you can read whatever is inside the file you can just copy paste the content and that would be the same as downloading it.
I really don't see the point and how could this be implemented.

**Reply by Boy132** [reply]
Reading and downloading are the same. If you can read a file you can download it and vice versa.
There won't be separate permissions for that.
